python/example_code/rds/hello/hello_rds.py

```diff
@@ -0,0 +1,33 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# snippet-start:[python.example_code.rds.hello]
+"""
+Purpose
+
+Shows how to use the AWS SDK for Python (Boto3) with the Amazon Relational Database Service
+(Amazon RDS) to list the databases in your account.
+"""
+
+import boto3
+from botocore.exceptions import ClientError
+
+# Create an RDS client
+rds_client = boto3.client("rds")
+
+try:
+    # Get a list of DB instances
+    response = rds_client.describe_db_instances()
```

Comment:
Same question on all of these that have paginators that aren't used here.

Reply:
Pagination fixed.